commands/sync.go

```diff
@@ -20,6 +21,15 @@ func (command *SyncCommand) Execute(args []string) error {
 	if err != nil {
 		return err
 	}
+	info, err := target.Client().GetInfo()
+	if err != nil {
+		return err
+	}
+
+	if info.Version == version.Version {
+		fmt.Printf("Good news! You already have the appropriate fly binary for this target.\n")
```

Comment:
minor UX thing: this should probably just say `version already matches; skipping`

we tend to stick with terse lowercase text elsewhere